### PR TITLE
otp component

### PR DESCRIPTION
**Pull request by RoopaSowmya14**

### CODE REVIEW

File: ui-app/client/src/components/Otp/OtpStyle.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+export default function OtpStyle({ theme }: { theme: Map<string, Map<string, string>> }) {
+	const css =
+		`
+		${PREFIX} {
+			display: flex;
+			
+		}
+
+		${PREFIX} ._inputBox {
+			width: 30px;
```
Comment (himanshu9489) [inline ui-app/client/src/components/Otp/OtpStyle.tsx:15]: write width and height properties in the styleProperties

File: ui-app/client/src/components/Otp/OtpStyle.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		${PREFIX} {
+			display: flex;
+			
+		}
+
+		${PREFIX} ._inputBox {
+			width: 30px; 
+			height: 30px; 
+			margin: 8px;
+			font-size: 16px;
```
Comment (himanshu9489) [inline ui-app/client/src/components/Otp/OtpStyle.tsx:18]: width, height, margin, font-size all should be in the styleProperties not in this one

File: ui-app/client/src/components/Otp/OtpStyle.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+		}
+
+		${PREFIX} ._inputBox {
+			width: 30px; 
+			height: 30px; 
+			margin: 8px;
+			font-size: 16px; 
+			text-align: center;
+			outline: none; 
+			border-style:solid;
```
Comment (himanshu9489) [inline ui-app/client/src/components/Otp/OtpStyle.tsx:21]: should be in styleProperties if there is not an specific reason for it to be here.

**Review by serialcoder13 — COMMENTED** [review]
LGTM

**Pull request merged**